input/sample_file.py

```diff
@@ -0,0 +1,28 @@
+"""Test case for imports."""
```

Comment:
Sample files for use in test cases should either be stored in strings, or in a folder inside the `tests/` folder. This separates them from your implementation, ensuring there isn't any confusion for users of your program.

Reply:
@RyanHiltyAllegheny to fix folder organization